core/src/main/java/org/mule/runtime/core/MessagePropertiesContext.java

```diff
@@ -1,250 +0,0 @@
-/*
- * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
- * The software in this package is published under the terms of the CPAL v1.0
- * license, a copy of which has been included with this distribution in the
- * LICENSE.txt file.
- */
-package org.mule.runtime.core;
-
-import org.mule.runtime.api.message.NullPayload;
-import org.mule.runtime.api.metadata.DataType;
-import org.mule.runtime.core.api.MessageProperties;
-import org.mule.runtime.core.config.i18n.CoreMessages;
-import org.mule.runtime.core.metadata.TypedValue;
-import org.mule.runtime.core.util.CopyOnWriteCaseInsensitiveMap;
-import org.mule.runtime.core.util.MapUtils;
-import org.mule.runtime.core.util.ObjectUtils;
-
-import java.io.Serializable;
-import java.util.Set;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-/**
- * This object maintains case-sensitive inbound and outbound scoped messages properties.
- * <ol>
- * <li> {@link PropertyScope#INBOUND} Contains properties that were on the message when
- * it was received by Mule. This scope is read-only.</li>
- * <li>{@link PropertyScope#OUTBOUND} Any properties set in this scope will be attached
- * to any outbound messages resulting from this message. This is the default scope.</li>
- * </ol>
- */
-public class MessagePropertiesContext implements MessageProperties, Serializable
```

Comment:
It's moved inside MuleMessageImplementation. We need to clean it up, but for now it's moved private/inner.